experimental/benchmarks/node-dc-eis/kill_node_linux

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
not related to this PR, but we should have common scripts in single place instead of having it every benchmark folder.